src/components/NpiLevelTypography.tsx

```diff
@@ -15,16 +15,24 @@ const ScientificValueMapper: ScientificValueMapperType = {
 
 interface NpiLevelTypographyProps extends TypographyProps {
   scientificValue?: ScientificValue;
+  publisherId?: string;
```

Comment:
Antar at samme skal gjøres for tiddskrift og serie også?
```suggestion
  channelId?: string;
```